pkg/codegen/testing/test/testdata/azure-native-nested-types/nodejs/types/output.ts

```diff
@@ -5,33 +5,4 @@ import * as pulumi from "@pulumi/pulumi";
 import * as inputs from "../types/input";
 import * as outputs from "../types/output";
 
-export namespace documentdb {
```

Comment:
Lovely. So the `export *` replacement does the same thing as before? This looks right.

Reply:
Yeah, `export * as ns from "file";` creates a namespace and re-exports everything exported by `file`. I read about it in the [v3.8 release notes](https://devblogs.microsoft.com/typescript/announcing-typescript-3-8/#export-star-as-namespace-syntax):

```typescript
export * as utilities from "./utilities";
```

is a syntactic sugar for:

```typescript
import * as utilities from "./utilities";
export { utilities };
```